java/pom.xml

```diff
@@ -384,11 +383,13 @@
                                     <arg value="-DCMAKE_EXPORT_COMPILE_COMMANDS=${CMAKE_EXPORT_COMPILE_COMMANDS}"/>
                                     <arg value="-DCUDF_CPP_BUILD_DIR=${CUDF_CPP_BUILD_DIR}"/>
                                     <arg value="-DGPU_ARCHS=${GPU_ARCHS}"/>
+                                    <arg value="-DCMAKE_CUDA_ARCHITECTURES=${GPU_ARCHS}"/>
                                 </exec>
                                 <exec dir="${native.build.path}"
                                       failonerror="true"
-                                      executable="make">
-                                    <arg value="-j"/>
+                                      executable="cmake">
+                                    <arg line="--build ."/>
+                                    <arg line="-j ${env.PARALLEL_LEVEL}"/>
```

Comment:
When I built before cleaning with this in the conda env supplying cmake 3.18.4, I get:
```
     [exec] Unknown argument ${env.PARALLEL_LEVEL}
     [exec] Usage: cmake --build <dir> [options] [-- [native-options]]
```

Reply:
If 3.18.5 fixes it maybe we can make it the minimum?

Reply:
Personally I don't think the Java bindings should require a cmake version beyond what is required for the rest of the cudf project.  Is there not a way to convey the build parallelism with the cmake version we already have?

Reply:
we currently have somewhat [inconsistent definitions](https://github.com/rapidsai/cudf/search?q=cmake_minimum_required) `cmake_minimum_required` in the repo.

it's not just about specifying the parallelism but also the ability to use different generators later on.

Reply:
> also the ability to use different generators later on.

I don't see this as a pressing need to warrant requiring a cmake version beyond anything else currently in the project.

Reply:
On a related note, it looks like the project will be moving to 3.18 shortly.  See https://github.com/rapidsai/cudf/pull/7107#discussion_r579376674.